metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -105,6 +107,20 @@ object UserConfiguration {
         |`src/main/scala:: src/main/java::`. Syntax such as `src/{main,test}::`
         |is not supported.
         |""".stripMargin
+    ),
+    UserConfigurationOption(
+      "bloop-generate-sbt",
+      """true""",
+      "true",
+      "Generate Bloop plugin file",
+      """Option that can specified to false if the user doesn't want to generate an sbt file adding the Bloop sbt plugin."""
+    ),
+    UserConfigurationOption(
+      "bloop-plugin-version",
```

Comment:
What about?
```suggestion
      "bloop-version",
```

Reply:
The user might think it will also change the Bloop server version, and that should be done manually by them. 

Or maybe that setting should also apply to the server version? What do you think?

Reply:
Makes sense

Reply:
Ok, so now it's possible to change Bloop version and restart the server when the setting change.

I think that's a pretty neat feature especially if a user wants to leverage new Bloop and Metals wer enot yet released. What do you think?